Review ch1 (#548)
worldbank/dime-data-handbook: chapters/1-reproducibility.tex

and \textbf{OSF}.\sidenote{\url{https://osf.io/registries}}
They all have different target audiences and features,
so select one that is appropriate to your work.
Study registration should be accessible to all projects,
although the amount of detail that it is possible at this stage can vary widely,

> depending on the timeline and complexity of the project.

Reviewer: what do you mean by 'accessible' here? seems odd to me. should be feasible for? or should be completed by? 
also 'the amount of detail that it is possible at this stage' doesn't work grammatically, do you mean, 'the amount of detail that can be provided at this stage'?

Author: ```suggestion
Study registration should be feasible for all projects,
as registries are typically free to access and can be completed with minimal information about the project.
A generally-acceptable practice will be to gradually revise and expand the level of detail in a registration over time,
adding more information as the planning for the project progresses.
```

How about this

of concrete measures that could be argued to measure to an abstract concept;
future choices about sample inclusion or exclusion;
or decisions about how to construct derived indicators.

> There are a variety of templates and checklists of details to include.\sidenote{

Reviewer: ```suggestion
There are a variety of templates and checklists of details to include.\sidenote{
We recommend this checklist: 
```

% Tools for transparency: GitHub, OSF
Email, however, is \textit{not} a documentation service, because communications are rarely well-ordered,\index{email}
can be easily deleted, and are not available for future team members.
At the vey least, emails and other decision-making communications
need to be archived and preserved (as, say, PDFs)

> so that they can be read easily by others in the future.

Reviewer: ```suggestion
in an organized manner so that they can be easily accessed and read by others in the future.
```

> There are also various software solutions for building proper documentation over time.

Reviewer: give an example? or do we have any examples on the wiki?

Author: I thought we had just cut out mentions of things like GitHub issues and dropbox paper. I could add Asana or something? Alternatively we could add "project management" on Wiki but I don't think we have a ton to say about how to manage tasks internally?

Reviewer: Ah, fair enough, we did cut those. Ok, i think it's fine to leave as it was.

data at this stage may still need to be embargoed
or have other, potentially permanent, access restrictions,
so you can instruct the archive to formally release the data later.
If your planned analysis requires the use of unpublishable data,
that data should always remain encrypted and be stored separately
so it is clear what portions of the code will work with and without

> obtaining a license to the needed restricted-access data.

Author: ```suggestion
obtaining a license to the needed restricted-access data.\sidenote{
  Details on how to document this type of material can be found at
  \url{https://doi.org/10.5281/zenodo.4319999}.}
```

with directly identifying data and questions of data ownership
before uploading raw data to any third-party server, whether public or not;\index{data ownership}
this is a legal question for your home organization.
If essential data must be placed under restricted use or restricted access,
develop a plan for storing that portion of the data
separately from publishable information,
as well as a protocol for allowing access in the future

> for replicators or reviewers under appropriate access agreements.

Reviewer: is this the response to the comment about archiving data? i think it could be a little clearer, like it's not necessarily going to make sense to separately save the identified data if that is an integral part of the analysis dataset. also, 'essential' here is confusing, i assume that you mean data required for the analysis, and it would be clearer to simply say that. 
this applies equally to data that cannot be published because it has PII and data that cannot be published based on the ownership terms, should be more explicit about this in the paragraph.

Reviewer: Additionally, i do not think we have sufficiently addressed comment 3.1.4, how to maximize transparency and reproducibility within the constraints of restricted use data that cannot be published for legal reasons or research designs with PII necessary for analysis. we say in the matrix, "Our recommendation is that all the data that can be published be published, and when data cannot be published, researchers should try to publish as much metadata as allowed, including information on how the data was obtained, what fields it contains and aggregations or descriptive statistics. Even if the data cannot be published, it is very rare for codes to contain restricted information or be barred from publication, so the code should still be made available. Finally, we recommend that reproducibility efforts be considered when designing the IRB and data licensing agreement, so there are specific conditions (such as a secure server or cold room) under which representatives from journals may access the data for the sole purpose of independently verifying results.", we should add all that (not sure that here is the exact right spot but within this section)

Author: ```suggestion
If data that is required for analysis must be placed under restricted use or restricted access,
including data that can never be distributed directly by you to third parties,
develop a plan for storing that data separately from publishable information.
This will allow you to maximize transparency by having a clear release package
as well as providing instructions or developing a protocol for allowing access in the future
for replicators or reviewers under appropriate access agreements.
```

Revised for first comment, going to insert the second comment language elsewhere

Author: Done in 570dd009575622c36ef17c65abfa5feaee0a3c1d

with directly identifying data and questions of data ownership
before uploading raw data to any third-party server, whether public or not;\index{data ownership}
this is a legal question for your home organization.
If data that is required for analysis must be placed under restricted use or restricted access,
including data that can never be distributed directly by you to third parties,
develop a plan for storing that data separately from publishable information.
This will allow you to maximize transparency by having a clear release package
as well as providing instructions or developing a protocol for allowing access in the future

> for replicators or reviewers under appropriate access agreements.

Author: ```suggestion
for replicators or reviewers under appropriate access agreements.\sidenote{
  Details on how to document this type of material can be found at
  \url{https://doi.org/10.5281/zenodo.4319999}.}
```
